chrome/locale/en-US/zotero/zotero.dtd

```diff
@@ -272,8 +272,10 @@
 <!ENTITY zotero.feedSettings.title.label                "Title:">
 <!ENTITY zotero.feedSettings.refresh.label1             "Update feed every">
 <!ENTITY zotero.feedSettings.refresh.label2             "hour(s)">
-<!ENTITY zotero.feedSettings.cleanupAfter.label1         "Remove read articles after ">
+<!ENTITY zotero.feedSettings.cleanupAfter.label1         "Remove articles after ">
 <!ENTITY zotero.feedSettings.cleanupAfter.label2         "day(s)">
+<!ENTITY zotero.feedSettings.cleanupReadAfter.label1         "Remove read articles after ">
+<!ENTITY zotero.feedSettings.cleanupReadAfter.label2         "day(s)">
```

Comment:
But isn't the important thing just to make sure there's an expiration for both? If somebody wants to keep read longer than unread, it's not really our problem. And adding a whole other check in the GUI for setting one below the other seems much more complicated and awkward. And "Do not retain…" is much less natural than "Remove unread…".